extensions/typescript-language-features/src/extension.browser.ts

```diff
@@ -108,7 +108,7 @@ async function preload(logger: Logger): Promise<void> {
 	}
 
 	const workspaceUri = vscode.workspace.workspaceFolders?.[0].uri;
-	if (!workspaceUri || workspaceUri.scheme !== 'vscode-vfs' || workspaceUri.authority !== 'github') {
+	if (!workspaceUri || workspaceUri.scheme !== 'vscode-vfs' || !workspaceUri.authority.startsWith('github')) {
```

Comment:
@joyceerhl Should I remove the authority check entirely and instead always try to call `remoteHubApi.loadWorkspaceContents` with the workspace url?